roles/jws/tasks/fastpackage.yml

```
@@ -1,19 +1,7 @@
 ---
-- name: Check arguments
-  ansible.builtin.assert:
-    that:
-      - package_name is defined
-    quiet: true
-
-- name: "Install package"
-  block:
-    - name: "Test if package {{ package_name }} is already installed"
-      ansible.builtin.command: rpm -q {{ package_name }}
-      register: rpm_info
-      changed_when: rpm_info.failed
-  rescue:
-    - name: "Install package {{ package_name }} using yum if missing"
-      ansible.builtin.yum:
-        name: "{{ package_name }}"
-        state: installed
-      when: rpm_info.failed
+- name: "Ensure {{ package_name }} is installed"
+  ansible.builtin.yum:
+    name: "{{ package_name }}"
+    state: installed
+  when:
+    - package_name is defined
```

@dsoumis That may seems weird, but rpm -q is WAY faster than yum to ascertain if a package is installed. Also, I would like to keep the assert: because if fastpackage is used, the package_name should be provided. Allowing fast package to be called with no package name (and silently skipping) is not IMHO a good behavior.

---

we have a much newer version of fastpackages in other collection, we need to port it here (it executes a single rpm command with the whole package list)

---

this: https://github.com/ansible-middleware/infinispan/blob/main/roles/infinispan/tasks/fastpackages.yml
it takes the list directly, does not need to be called in a loop